### DISCUSSION TITLE
Picture fo Measurements overview

### DISCUSSION THREAD

**Discussion by Slylele** [freesewing/freesewing#1624] [Forum]
maybe we could propose an overview of all measurements on the measurements home page
I've started to prepare it quickly (it's uggly but it's just quick draft to see what could be the result...)
![Measurements_Overview](https://user-images.githubusercontent.com/57619777/79065814-68f37f80-7cb3-11ea-8440-9c86dbc6a6a8.png)

We even could imagine further improvements linked to this (here some quick ideas):
- [ ] to link it to the measurements for each pattern (I don't know if we can set each measurement as parameter in the svg file so that depending on the pattern it's displayed or not)... For example on any pattern "Required measurements" page, we could display only the measures needed
- [ ] Set link directly from the measue on the picture to the explanation page of the measure
- [ ] on persons measures page, change color or set a mark to identify the measures filled or not (and maybe display the measure value?)

**Comment by joostdecock** [comment]
This is still a great idea, but it's also very stale. So I'm converting this to an idea, so that we can make it part of our efforts to build a new freesewing.org site.
